pycheribuild/projects/disk_image.py

```diff
@@ -1069,6 +1069,236 @@ def cheribsd_class(self):
         return self._source_class
 
 
+class BuildBesspinCheriBSDDiskImage(BuildDiskImageBase):
+    project_name = "disk-image-besspin"
+    _source_class = BuildCHERIBSD
+    disk_image_prefix = "cheribsd-besspin"
+    include_boot = True
+
+    @classmethod
+    def setup_config_options(cls, **kwargs):
+        super().setup_config_options(default_hostname=_default_disk_image_hostname("cheribsd-besspin"),
+                                     extra_files_suffix="-besspin", **kwargs)
+        cls.strip_binaries = cls.add_bool_option(
+            "strip", default=True, help="strip ELF files to reduce size of generated image")
+        cls.use_cheribsd_purecap_rootfs = cls.add_bool_option(
+            "use-cheribsd-purecap-rootfs", default=True, help="Use the rootfs built by cheribsd-purecap instead")
```

Comment:
Do we really need this? It's a silly target to build, just leave it as hybrid and thus uninteresting?

Reply:
I'll remove this. It's leftover from foking the minimal target.

Reply:
... we should probably remove that for the same reason (though at least that one is default=False)